src/build/env.py

```diff
@@ -116,7 +116,15 @@ def __enter__(self):  # type: () -> IsolatedEnvironment
             if path in sys_path:
                 sys_path.remove(path)
 
-        self._replace_env('PATH', self._get_env_path('scripts'))
+        env_scripts = self._get_env_path('scripts')
+        if env_scripts is None:  # pragma: no cover
+            raise RuntimeError('Missing scripts directory in sysconfig')
```

Comment:
Can this be true? Do we need to raise for it?

Reply:
It should never be true, but I am making mypy happy. The possibility of a Python implementation not having a `scripts` directory is pretty slim, the only use cases I would imagine is something like MicroPython or Python for WebAssembly, but this package wouldn't be relevant there.

Reply:
you could also just use the typing.cast operation to cast away the None 🤷 but is good this way too 👍 the downside is no coverage for it though...

Reply:
I don't really care about the coverage of this particular check, hence the `# pragma: no cover`. I could use `typing.cast`, this just felt better and since it was such a simple implementation, I went for it. If it was more complex, I would probably have opted the other way :blush:.